doc/releases/release-notes-4.0.rst

```diff
@@ -384,6 +438,12 @@ Drivers and Sensors
 * DAC
 
   * DAC API now supports specifying channel path as internal. Support has been added in STM32 drivers.
+  * Updated the NXP counter_mcux_lptmr driver to support multiple instances of the lptmr
```

Comment:
Sorry I missed this- these look like they should be in the "Counter" section, not DAC